bot.js

```diff
@@ -123,9 +123,10 @@ if (!process.env.clientId || !process.env.clientSecret) {
   require(__dirname + '/components/plugin_glitch.js')(controller);
 
   var normalizedPath = require("path").join(__dirname, "skills");
-  require("fs").readdirSync(normalizedPath).forEach(function(file) {
-    require("./skills/" + file)(controller);
-  });
+  require("fs").readdirSync(normalizedPath)
+.filter(file => file !== '.DS_Store').forEach(function(file) {
```

Comment:
Whitespace updates that you probably didn't intend.

Reply:
Thanks for pairing with me! I was calling out the missing indentation.

There is also the method of filtering as we discussed. What do we do if another file get's added... `.gitignore` for example? This _exclusionary_ list is "unbounded". We know that we're `require`ing the files we find. That means they must all be valid javascript files. Our convention in this ecosystem is to name those `.js`. Could we lean into that convention and be _inclusionary_ here.

`filter` means to _exclude_ elements in an array/enumerable.
[`find`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) is the means to _include_ elements in an array/enumerable.

Reply:
`require("fs").readdirSync(normalizedPath).filter(file => file.endsWith('.js')).forEach(function(file) {

  require("./skills/" + file)(controller);
});` This still uses filter but is inclusive. I tried find but it wasn't working as expected.